src/cli/cli_coap_secure.cpp

```diff
@@ -147,6 +173,42 @@ template <> otError CoapSecure::Process<Cmd("set")>(Arg aArgs[])
     return error;
 }
 
+/**
+ * @cli coaps start
+ * @code
+ * coaps start
+ * Done
+ * @endcode
+ * @code
+ * coaps start false
+ * Done
+ * @endcode
+ * @code
+ * coaps start 8
+ * Done
+ * @endcode
+ * @cparam coaps start [@ca{check-peer-cert} | @ca{max-conn-attempts}]
+ * The `check-peer-cert` (peer certificate check) and `max-conn-attempts`
+ * (maxinum number of allowed connection attempts) parameters work
+ * together in the following combinations, even though you can only specify
+ * one argument:
+ *   * Argument omitted:
```

Comment:
Does this mean both arguments are omitted?

Can we maybe rework these bullets to note what the defaults are, rather than saying "if you omit this, the value is x"